### DISCUSSION TITLE
Importing gds files

### DISCUSSION THREAD

**Discussion by strawberrykim** [ansys/pyaedt#2052] [Q&A] [answered]
Hello,

I was wondering how gds files can be imported through pyaedt in hfss? I usually go Modeler->Import->Select Layers, but I do not see a function for doing this in pyaedt. Thank you for your help.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Please see this example, it imports GDS in a EDB file, then you just need to open the EDB file in HFSS3DLayout:

https://aedt.docs.pyansys.com/version/stable/examples/00-EDB/10_GDS_workflow.html
